### ISSUE TITLE
0.17.0-rc1: ResourceMgr defaults sabotage DHT routing in Desktop/Brave nodes

### ISSUE THREAD

**Issue by lidel** [ipfs/kubo#9406]
### Version

```Text
0.17.0-rc1
```

### Config

```json
> ipfs config --json Swarm.ConnMgr # these are values used in Kubo node managed by Brave
{
  "GracePeriod": "20s",
  "HighWater": 40,
  "LowWater": 20,
  "Type": "basic"
}

$ ipfs config --json Experimental.AcceleratedDHTClient
false
```

### Description

IPFS Desktop and Brave run with lowered ConnMgr settings such as  HighWater and LowWater.

Running 0.17.0-rc1 with 

```json
"HighWater": 40,
"LowWater": 20,
```

and opening `/ipns/en.wikipedia-on-ipfs.org/wiki/`  on empty repo produces  a wall of errors and seems to break DHT-based content routing (bitswap works, but content that requires DHT rarely loads, browser timeouts).

`ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 10619 times with error "system: cannot reserve connection: resource limit exceeded".`

<details>
  <summary>Click to expand ERROR logs printed to stdout</summary>

```console
Daemon is ready
# browsing /ipns/en.wikipedia-on-ipfs.org/wiki/ 
2022-11-14T17:36:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 1004 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:36:49.087Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:36:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 7811 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:36:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:37:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 67 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:37:19.087Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:37:29.085Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 1655 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:37:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:37:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 3188 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:37:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:37:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 8115 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:37:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:37:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4446 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:37:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:38:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 10619 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:38:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:38:19.087Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 3482 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:38:19.087Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:38:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 6703 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:38:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:38:39.085Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 14239 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:38:39.085Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:38:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 10891 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:38:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:38:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 10468 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:38:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
# opened ipns://docs.ipfs.tech
2022-11-14T17:39:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 2206 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:39:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:39:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 7171 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:39:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:39:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 1220 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:39:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:39:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 2861 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:39:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:39:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4875 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:39:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:39:59.087Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 8489 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:39:59.087Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:40:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 2231 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:40:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:40:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 311 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:40:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:40:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 1697 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:40:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:40:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 2059 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:40:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:40:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 5566 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:40:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:40:59.089Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 9108 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:40:59.089Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:41:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 5976 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:41:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:41:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 9109 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:41:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:41:29.085Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 3831 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:41:29.085Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:41:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 8009 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:41:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:41:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 858 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:41:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:41:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4082 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:41:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:42:09.085Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 8273 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:42:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:42:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4756 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:42:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:42:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 2221 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:42:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:42:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 47 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:42:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:42:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 39 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:42:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:42:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 3728 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:42:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:43:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 922 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:43:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:43:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4107 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:43:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:43:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 2281 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:43:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:43:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 5868 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:43:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:43:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 2705 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:43:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:43:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 5769 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:43:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:44:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 5276 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:44:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:44:19.085Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 7376 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:44:19.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:44:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 1142 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:44:29.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:44:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4675 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:44:39.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:44:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 488 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:44:49.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:44:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4260 times with error "system: cannot reserve connection: resource limit exceeded".
2022-11-14T17:44:59.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:57	Consider inspecting logs and raising the resource manager limits. Documentation: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
2022-11-14T17:45:09.086Z	ERROR	resourcemanager	libp2p/rcmgr_logging.go:53	Resource limits were exceeded 4059 times with error "system: cannot reserve connection: resource limit exceeded".
```

</details>

### How to reproduce 

1. Install 0.17.0-rc1, make sure repo is empty
1. Set "HighWater": 40 and   "LowWater": 20
1. Restart node
1. Try browsing websites **that are not cached locally**  (try using `/ipns/en.wikipedia-on-ipfs.org/wiki/`  – walk from article to article)
1. Observe stdout / logs + the delay in initial page loads (+ occasional timeout in browser)

### How to fix?

- I feel we need to create `desktop` profile that is fine-tuned for Desktop and Brave, and switch desktop use cases  to that profile.
  - Perhaps it should  have `HighWater` set bit higher? (iiuc we limit connection number based on this value) so we dont sabotage DHT walk and avoid ERRORs in logs? 

cc @ipfs/gui-dev  for visibility and suggestions

**Comment by BigLep** [comment]
@lidel: you listed 

```
$ ipfs config --json Experimental.AcceleratedDHTClient
true
```

Is Experimental.AcceleratedDHTClient enabled?  If so, I expect that is the culprit as that will quickly consume connections.

If a HighWater is set, we set the system scope's maxConns to 2*HighWater: https://github.com/ipfs/kubo/blob/254d81a9d5595c3e637c7573d56125836d5f5055/core/node/libp2p/rcmgr_defaults.go#L203

**Comment by BigLep** [comment]
@ipfs/kubo-maintainers : can we please get others to reproduce this so we can figure out our path forward on 2022-11-14 during standup.

**Comment by ajnavarro** [comment]
This should be fixed with #9407 if we are using acceleratedDHT on that config too.
